TestMatrixMultiplication/UnitTest1.cs

```diff
@@ -0,0 +1,74 @@
+using System.Diagnostics;
+using MatrixMultiplication;
+
+namespace TestMatrixMultiplication
+{
+    public class Tests
+    {
+        [Test]
+        public void TestMultiplicateMatrix()
+        {
+            Matrix matrix1 = new ("testMatrix1.txt");
+            Matrix matrix2 = new ("testMatrix2.txt");
+            var result = matrix1.MultiplicateMatrix(matrix2);
+            result.WriteMatrixFile("testresult.txt");
+            Assert.That(File.ReadAllText("testResult.txt"),
+                Is.EqualTo(File.ReadAllText("testRightResult.txt")));
+        }
+
+        [Test]
+        public void TestMultiThreadMultiplicateMatrix()
+        {
+            Matrix matrix1 = new("testMatrix1.txt");
+            Matrix matrix2 = new("testMatrix2.txt");
+            var result = matrix1.MultiThreadMultiplicateMatrix(matrix2);
+            result.WriteMatrixFile("testresult.txt");
+            Assert.That(File.ReadAllText("testResult.txt"),
+                Is.EqualTo(File.ReadAllText("testRightResult.txt")));
+        }
+
+        [Test]
+        public void TestInvalidMatrix()
+        {
+            Matrix matrix = default!;

```

Comment:
```suggestion
            Matrix matrix = null!;
```
default хорошо, когда мы не знаем значения по умолчанию для типа (например, внутри генерика).